compiler/src/iree/compiler/Dialect/Stream/Transforms/Passes.cpp

```diff
@@ -106,9 +112,15 @@ void buildStreamTensorPassPipeline(OpPassManager &passManager,
     passManager.addPass(IREE::Stream::createAnnotateAffinitiesPass());
   }
 
+  assert((!clReplicateGlobalsPerAffinity || !clTransferGlobalsPerAffinity) &&
+         "Cannot replicate and transfer at the same time.");
   if (clReplicateGlobalsPerAffinity) {
     passManager.addPass(IREE::Stream::createReplicateGlobalsPerAffinityPass());
   }
+  if (clTransferGlobalsPerAffinity) {
+    passManager.addPass(IREE::Stream::createReplicateGlobalsPerAffinityPass(
+        IREE::Stream::ReplicateGlobalsPerAffinityPassOptions{true}));
+  }
```

Comment:
You can pass `clTransferGlobalsPerAffinity` to `createReplicateGlobalsPerAffinityPass`. I think they can be both true. The difference is just about which mode it is picking up. E.g.,

```
if (clReplicateGlobalsPerAffinity) {
passManager.addPass(IREE::Stream::createReplicateGlobalsPerAffinityPass(
        IREE::Stream::ReplicateGlobalsPerAffinityPassOptions{clTransferGlobalsPerAffinity}));
}
```